### PR TITLE
Standardized the usage docs for cf-forms

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updated usage documentation for cf-forms to match the formatting and structure of #341 and #347.

## Additions

- Table of Contents

## Changes

- Standardized headings and org of description text

## Testing

- Test on your local markup preview tool

## Review

- @KimberlyMunoz 
- @contolini 
- @cfpb/front-end-team-admin 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-forms/usage.md [hunk 1]
```diff
@@ -1,12 +1,37 @@
-# Forms
+The cf-forms component provides extensions to the basic form styles for Capital Framework.
```
Comment (KimberlyMunoz) [inline src/cf-forms/usage.md]: Add a line for dependencies and remove the dependencies section.

**Comment by jimmynotjim** [comment]
@KimberlyMunoz Ready for review again

File: src/cf-forms/usage.md (hunk 1 above)
Comment (KimberlyMunoz) [inline src/cf-forms/usage.md]: Resolved!

File: src/cf-forms/usage.md [hunk 2]
```diff
@@ -1,12 +1,34 @@
...
+
+[`cf-core`](../cf-core), ['cf-buttons'](../cf-buttons), and ['cf-icons'](../cf-icons) components are all dependencies of this component."
```
Comment (KimberlyMunoz) [inline src/cf-forms/usage.md]: It looks like only `cf-core` is in tick marks and the rest is in quotes.

File: src/cf-forms/usage.md [hunk 3]
```diff
@@ -1,12 +1,34 @@
...
   be sure to run the file through
   [Autoprefixer](https://github.com/postcss/autoprefixer),
   or your compiled Capital Framework CSS will
   not work perfectly in older browsers.
 
-## Vars
+## Table of contents
+
+- [Variables](#variables)
+    - [Color variables](#color-variables)
```
Comment (KimberlyMunoz) [inline src/cf-forms/usage.md]: This is minor, but can you include the line about the colors being from the U.S. Web Design Standards and comments with the colors to be consistent with the rest?

**Comment by KimberlyMunoz** [comment]
Almost there 🚂

**Comment by jimmynotjim** [comment]
@KimberlyMunoz ready again.

**Comment by KimberlyMunoz** [comment]
:+1:

**Pull request merged**